build/config/compiler/BUILD.gn

```diff
@@ -642,14 +647,35 @@ if (is_win) {
     "/wd4311",  # Pointer truncation from PVOID to DWORD.
     "/wd4477",  # Format string requires wchar_t*
   ]
-  import("//build/toolchain/goma.gni")
-  if (is_clang && use_goma) {
-    # goma compilation doesn't preserve file case, so don't warn about
-    # case-mismatches.
-    default_warning_flags += [
+
+  if (is_clang) {
+    # These warning flags seem to be Clang specific in addition to those
+    # specified for MSVC above.
+    extra_warning_flags = [
+      # Warnings as errors.
+      "-Werror",
+
+      # Some third party dependencies like ICU add flags that are not supported
+      # by clang-cl yet.
+      "-Wno-unknown-argument",
+
       "-Wno-nonportable-include-path",
       "-Wno-nonportable-system-include-path",
```

Comment:
Why are you turning these on more often? The are working around a goma bug.

Reply:
I ran into these on my non-corp workstation without goma setup.